calendars/numpy.yaml

```
@@ -56,11 +56,26 @@ events:
 
       To add to the meeting agenda the topics you’d like to discuss,
       follow the link: https://hackmd.io/3f3otyyuTte3FU9y3QzsLg
-    begin: 2022-02-10 14:00:00 +00:00
-    end: 2022-02-10 15:00:00 +00:00
+    begin: 2022-11-03 12:00:00 +00:00
+    end: 2022-11-03 13:00:00 +00:00
     url: https://us06web.zoom.us/j/82563808729?pwd=ZFU3Z2dMcXBGb05YemRsaGE1OW5nQT09
     repeat:
       interval:
         # seconds, minutes, hours, days, weeks, months, years
-        days: 14
+        days: 28
+      until: 2025-01-01 00:00:00 +00:00
+      
+      - summary: NumPy Newcomers' Hour
```

I suspect the indentation here is the cause of the parsing error:

```suggestion
  - summary: NumPy Newcomers' Hour
```